docs/web/api-commands-operations-events.mdx

```diff
@@ -1,26 +1,15 @@
----
-title: Commands, Operations and Events Reference
-description: VertiGIS Studio Web - Commands, Operations and Events API Reference
----
-
-import ViewerMessaging from "../../src/components/ViewerMessaging";
-
-Commands and operations are runnable, independent units of work within VertiGIS Studio Web. [Commands](#commands) are units of work which may take an input but do not produce an output, while [operations](#operations) differ because they produce output. You can learn more about commands and operations in our [conceptual documentation](configuration-commands-operations.mdx)
-
-[Events](#events) are distinct, discrete messaging about a change or update that occurs in the viewer. They can be used to initiate behaviors, communicate data changes, and more. Events themselves do not execute behavior, but merely provide messaging so subscribers can act on changes. You can learn more about events in our [conceptual documentation](sdk-events-overview.mdx)
-
-## Commands
-
-<ViewerMessaging product="web" type="command" />
-
-## Operations
-
-<ViewerMessaging product="web" type="operation" />
-
-## Events
-
-<ViewerMessaging product="web" type="event" />
-
-## Argument Definitions
-
-<ViewerMessaging product="web" type="argument" />
+import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
+
```

Comment:
I think we should restore these headers to something like this that will:
- Show something formatted if the page is slow. Currently you see the filename as the title.
- Keep this page out of the sitemap
- Hide the edit link

```
---
title: Redirecting...
custom_edit_url: null
unlisted: true
---
```